### DISCUSSION TITLE
Logs for users actions in admin

### DISCUSSION THREAD

**Discussion by pnpaes** [craftcms/cms#14217] [Q&A]
Hi is there a way to collect logs specifically for when users access the admin panel?
I would aso like to keep track of user actions in the admin area (per user). is that possible?

**Comment by piotrpog** [comment]
I guess you could attach some code to this event:
https://github.com/craftcms/cms/blob/develop/src/events/AuthenticateUserEvent.php
And append some info to custom log file there.

**Comment by alexanderbuergin** [comment]
Maybe that one helps:
https://plugins.craftcms.com/activity?craft4
